test/e2e/helpers_test.go

```diff
@@ -151,7 +151,9 @@ func createKINDBuilder() *environments.Builder {
 	}
 	builder = builder.WithClusterBuilder(clusterBuilder)
 	builder = builder.WithAddons(metallb.New())
-	builder = builder.WithAddons(buildImageLoadAddons(imageLoad, kongImageLoad)...)
+	if shouldLoadImages() {
+		builder = builder.WithAddons(buildImageLoadAddons(controllerImageOverride, kongImageOverride)...)
```

Comment:
Given https://github.com/Kong/kubernetes-testing-framework/issues/440 won't this be problematic if `controllerImageOverride` is not specified (or vice versa)?

Reply:
Ah ok, I'm blind 🤦